app/helpers/search_results_helper.rb

```diff
@@ -1,4 +1,25 @@
 module SearchResultsHelper
+  def descriptions_with_other_handling(commodity)
+    ancestor_descriptions = commodity.ancestor_descriptions.reverse
+    return sanitize commodity.description.to_s unless commodity.description.to_s.match(/^other/i)
+
+    descriptions = []
+
+    ancestor_descriptions.each do |ancestor_description|
+      if ancestor_description.to_s.match(/^other/i)
+        descriptions.unshift(ancestor_description.to_s)
+      else
+        descriptions.unshift(ancestor_description.to_s)
+        break
+      end
+    end
+
+    descriptions[-1] = "<strong>#{descriptions.last}</strong>"
```

Comment:
this is OK, I just notice we often use `tag` here in the Frondent to generate HTML tags